server/gameconnection.py

```diff
@@ -305,11 +305,18 @@ def send_message(self, message):
         """
 
         async with db.engine.acquire() as conn:
+            result = await conn.execute(
+                """ INSERT INTO `moderation_report` (`reporter_id`, `game_id`, `game_incident_timecode`)
```

Comment:
Description with missing telling that it is an auto-generated report for a teamkill from clicking in the game.

Reply:
you're right!